src/release/main.d

```diff
@@ -60,7 +61,12 @@ void main ( string[] params )
 
     auto con = HTTPConnection.connect(getConf());
     auto repo = con.repository(getUpstream());
-    auto tags = repo.releasedTags().map!(a=>Version.parse(a.name)).array;
+
+    auto tags = repo
+        .releasedTags()
+        .map!(a=>Version.parse(a.name).ifThrown(Version.init))
```

Comment:
I'd much rather use `Nullable!Version` (or even algebraic like here https://github.com/sociomantic-tsunami/neptune/blob/master/src/overview/repository.d#L45) to indicate erroneous entries.